app/views/katello/api/v2/hosts_bulk_actions/erratum.json.rabl

```diff
@@ -5,3 +5,5 @@ node :applicable_hosts do |erratum|
   select(["#{::Host.table_name}.id", "#{::Host.table_name}.name"]).
   collect { |host| {:name => host.name, :id => host.id} }
 end
```

Comment:
i think you forgot to remove the actual hosts lists :)

Reply:
I thought we were going to merge this PR right after mine, which would get rid of it. https://github.com/Katello/katello/pull/7829/files